.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-## The main Github Actions workflow
+i## The main Github Actions workflow
```

Comment:
Vi accident?

Reply:
oh good catch - that may be why the workflow was never triggered. 
indeed it was vi

Reply:
updated in [d0de1af](https://github.com/stacks-network/stacks-core/pull/4765/commits/d0de1af451f941e2d49b18f19fc29d3cb6e90167)

and now the worfklow is running as well!